src/lang/Messages.properties

```diff
@@ -2037,6 +2037,7 @@ spider.api.action.setOptionMaxParseSizeBytes = Sets the maximum size, in bytes,
 spider.api.action.scan = Runs the spider against the given URL (or context). Optionally, the 'maxChildren' parameter can be set to limit the number of children scanned, the 'recurse' parameter can be used to prevent the spider from seeding recursively, the parameter 'contextName' can be used to constrain the scan to a Context and the parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
 spider.api.action.scanAsUser = Runs the spider from the perspective of a User, obtained using the given Context ID and User ID. See 'scan' action for more details.
 spider.api.action.setOptionScopeString = Use actions [add|modify|remove]DomainAlwaysInScope instead.
+spider.api.view.addedNodes = Returns a list of the names of the nodes added to the Sites tree by the specified scan.
```

Comment:
Well, its actually returning the 'fully qualified' node name, eg:

- http://localhost:8080/bodgeit/basket.jsp (quantity_18,update)

So it includes the full URL + the names of the post parameters